projects/storefrontlib/cms-components/product/product-list/product-grid-item/product-grid-item.component.html

```diff
@@ -26,7 +26,7 @@
       {{ 'productDetails.noReviews' | cxTranslate }}
     </div>
   </div>
-  <div class="cx-product-price-container">
+  <div *ngIf="!product.multidimensional" class="cx-product-price-container">
```

Comment:
`query:` could it be the case that customers can already have multidimensional products in their stores despite the lack of SPA support? If so (but I'm guessing it's unlikely) this `ngIf` statement could be a breaking change.